js/app.js

```diff
@@ -12,3 +12,195 @@ document.querySelectorAll('.menu-item').forEach((n) => {
+        <ul class="dev desk_dev">
+        <li class="upper dev_item font_p">${proFeatures[0]}</li>
+    <li class="dev_item"><div class="seperator"></div></li>
+    <li class="dev_item font_p">${proFeatures[1]}</li>
+    <li class="dev_item"><div class="seperator"></div></li>
+    <li class="dev_item font_p">${proFeatures[2]}</li>
+  
+  </ul>
+  <div class="desk_popup_image"> <img class="project_image" src="${proImage[0]}" alt="project 1" />
+  <img
+    class="project_image_des"
+    src="${proImage[1]}"
+    alt="project_1"
+  />
+  </div>
+  <div class="desk_popup_detail">
+  <div class="desk_popup_info">
+  <p class="project_details font_p">
+  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.  
+  </p>
+  </div>
+  <div class="desk_link_con">
+  <ul><li><ul class="api mob_menu">
+  <li class="api_item font_p mob_item">${proTech[0]}</li>
+  <li class="api_item font_p mob_item">${proTech[1]}</li>
+  <li class="api_item font_p mob_item">${proTech[2]}</li>
+  
+</ul></li>
+<li><ul class="api desk_menu">
+
+<li class="api_item font_p desk_item">${proTech[3]}</li>
+<li class="api_item font_p desk_item">${proTech[4]}</li>
+<li class="api_item font_p desk_item">${proTech[5]}</li>
+</ul></li></ul>
+  
+  <button type="button" class="btn link_button font_p">See Live<img class="desk_popup_icon" src="./images/icon.png" alt="icon" /></button>
+<button type="button" class="btn link_button font_p">See Source<img class="desk_popup_icon" src="./images/github.png" alt="github" /></button></div></div>
+</div>`;
+    const wrap = document.querySelector('.wrapper');
+    const overlay = document.createElement('div');
+    overlay.classList.add('j-popup');
+    const wbg = document.createElement('div');
+    wbg.classList.add('wbg-popup');
+    wbg.innerHTML = cardContent;
+    overlay.append(wbg);
+    wrap.append(overlay);
+    document.querySelector('.j-popup').addEventListener('click', () => {
+      overlay.classList.toggle('active');
+    });
+  });
+});
+
+// form validation
+
+const form = document.getElementById('form');
+const mail = document.getElementById('email');
+const message = document.querySelector('small');
+
+function showError() {
+  message.innerText = 'All letters in email must be in lowercase';
+  message.classList.add('error');
+}
+function validateEmail(email) {
+  const emailVal = email.value.trim();
+  const lowerCaseWords = emailVal.toLowerCase();
+  if (emailVal !== lowerCaseWords) {
+    return showError();
+  }
+  message.innerText = '';
+  return true;
+}
+
+form.addEventListener('click', (e) => {
+  e.preventDefault();
+  validateEmail(mail);
+});
```

Comment:
- [ ] If a user submits the contact form with the email field containing only lowercase characters the form should send normally but it is not sending. Kindly fix it.